src/components/Landing/Button/Button.styles.js

```diff
@@ -26,7 +26,7 @@ const StyledButton = styled.button`
   text-decoration: none;
 
   background-color: var(--btn-bg-color);
-  border: 2px solid ${colors.tuatara};
+  border: 2px solid var(--btn-color);
```

Comment:
Tu peux aussi faire
```suggestion
  border: 2px solid currentColor;
```
Ça marche aussi. 😉 Ça prend automatiquement la couleur du texte (ref. `color`)